e2e/features/address-validation.feature

```diff
@@ -6,11 +6,19 @@ Feature: Address details are validated
     And I complete the Home Office reference page
     And I complete the Basic details page
 
+  @create-case @alternate @RIA-688
+  Scenario: Unanswered fixed address question is not allowed
+
+    Given I do nothing
+    Then the Continue button is disabled
```

Comment:
it looks ok, although tests at this granularity could quickly become a drain on the testing time if we implement alot of them. Could it just be a `Given` on the existing test below it (`@RIA-732`)?

Reply:
Yes :)

Reply:
I'll do that